_modules/sesceph/__init__.py

```diff
@@ -161,14 +162,22 @@ def _update_partition(action, dev, description):
 
 
 
-def zap(dev):
+def zap(dev = None, **kwargs):
     """
     Destroy the partition table and content of a given disk.
     """
+    if dev is not None:
+        log.warning("Depricated use of function function use kwargs")
+    test_dev = kwargs.get("dev", None)
+    if test_dev != None:
+        dev = test_dev
```

Comment:
Wouldn't dev=kwargs.get("dev", None) accomplish the same?  Either dev is value or None.  If both dev arg and kwargs dev: value were specified, the kwarg is still preferred.